app/templates/components/fandom-app-smart-banner.hbs

```diff
@@ -0,0 +1,13 @@
+<div class="fandom-app-smart-banner__close" {{action 'close'}}>{{svg 'close' viewBox='0 0 18 18'}}</div>
+<a href="{{link}}" class="fandom-app-smart-banner__link">
+	<div class="fandom-app-smart-banner__info">
+		<img class="fandom-app-smart-banner__icon"
+			 src="https://services.wikia.com/static-assets/image/9b0dfb2e-9a8e-4c2d-ab24-232664ebe6d3">
+		<div class="fandom-app-smart-banner__meta">
+			<div class="fandom-app-smart-banner__title">FANDOM</div>
+			<span class="fandom-app-smart-banner__description">Get it free at the</span>
```

Comment:
I'd extract all strings to i18n

Reply:
let's do it when we decide to translate this message. Depending on language the store name may appear in different place of string, so it will probably require us to merge these strings into single message